### PR TITLE
Bump kubeadm package to kubernetes/kubernetes@b1d8961fe43015

### PR DESCRIPTION
**Pull request by errordeveloper**
I think we are quite close to have most major bugs fixed and a new unstable packages would help many of the users, let's cut this release and get folks to test it.

### CODE REVIEW

**Comment by errordeveloper** [comment]
@luxas @dgoodwin @mikedanese PTAL

File: rpm/kubelet.spec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 %global KUBE_VERSION 1.4.4
-%global KUBEADM_VERSION 1.5.0-alpha.1.409+714f816a349e79
+%global KUBEADM_VERSION v1.5.0-alpha.1.955+b1d8961fe43015
```
Comment (dgoodwin) [inline rpm/kubelet.spec]: Added "v" here looks wrong, will end up with a bad URL when fetching as it's still there as well.

Comment (errordeveloper) [inline rpm/kubelet.spec]: Well spotted, I'll fix it.

Comment (errordeveloper) [inline rpm/kubelet.spec]: Done.

**Comment by luxas** [comment]
LGTM

@dgoodwin approved it as well, so merging

**Comment by errordeveloper** [comment]
It doesn't look like this made it to the unstable RPM channel, neither did 1.4.4 bump make it there. But it's all in Debian repo already.

**Comment by luxas** [comment]
@mikedanese Does your auto-packaging script package debs or are that only for debs so far?

**Pull request merged**